hummingbot/connector/derivative/bybit_perpetual/bybit_perpetual_in_flight_order.py

```diff
@@ -0,0 +1,103 @@
+from decimal import Decimal
+from typing import (
+    Any,
+    Dict,
+    Optional, List,
+)
+
+from hummingbot.connector.in_flight_order_base import InFlightOrderBase
+from hummingbot.core.event.events import (
+    OrderType,
+    TradeType
+)
+
+
+class BybitPerpetualInFlightOrder(InFlightOrderBase):
+    def __init__(self,
+                 client_order_id: str,
+                 exchange_order_id: Optional[str],
+                 trading_pair: str,
+                 order_type: OrderType,
+                 trade_type: TradeType,
+                 price: Decimal,
+                 amount: Decimal,
+                 leverage: int,
+                 position: str,
+                 initial_state: str = "Created"):
+        super().__init__(
+            client_order_id,
+            exchange_order_id,
+            trading_pair,
+            order_type,
+            trade_type,
+            price,
+            amount,
+            initial_state,
+        )
+        self.fee_asset = self.quote_asset if self.quote_asset == "USDT" else self.base_asset
```

Comment:
Fee asset should depend on which side the trade is on right? If the order is a "buy" the fee asset should be the base, and vice versa.

Reply:
Not really. Bybit specifies that fees are charged on base token for non USDT perpetual market, and on quote token for USDT perpetual market.